src/code42cli/extensions/__init__.py

```diff
@@ -0,0 +1,49 @@
+from code42cli.click_ext.groups import ExtensionGroup
+from code42cli.main import CONTEXT_SETTINGS
+from code42cli.options import debug_option
+from code42cli.options import pass_state
+from code42cli.options import profile_option
+
+
+def sdk_options(f):
```

Comment:
Maybe you just want to be really explicit, but this could just be 

```
from code42cli.options import sdk_options as sdk_opts

def sdk_options(f):
    """Decorator that adds two `click.option`s (--profile, --debug) to wrapped command, as well as
    passing the `code42cli.options.CLIState` object using the [click.make_pass_decorator](https://click.palletsprojects.com/en/7.x/api/#click.make_pass_decorator),
    which automatically instantiates the `py42` sdk using the Code42 profile provided from the `--profile`
    option. The `py42` sdk can be accessed from the `state.sdk` attribute.

    Example:

        @click.command()
        @sdk_options
        def get_current_user_command(state):
            my_user = state.sdk.users.get_current()
            print(my_user)
    """
    f = sdk_opts()(f)
    return f
```

Reply:
Or even `sdk_options = sdk_opts`

Reply:
I mainly wanted to have the object we expose be it's own thing in case we ever want to change our internal `sdk_options` we're not having to work around the extension backwards compatibility.